api/internal/localizer/locloader.go

```diff
@@ -0,0 +1,382 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package localizer
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+	"path/filepath"
+	"strings"
+
+	"sigs.k8s.io/kustomize/api/ifc"
+	"sigs.k8s.io/kustomize/api/internal/git"
+	"sigs.k8s.io/kustomize/api/loader"
+	"sigs.k8s.io/kustomize/kyaml/errors"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+const dstPrefix = "localized"
+const localizeDir = "localized-files"
+const fatalPrefix = "should never happen"
+
+const errAlpha = "alpha unsupported localize feature"
+const errCleanDst = "unable to clean localize destination"
+const errReference = "invalid localize reference location '%s'"
```

Comment:
IIRC we removed the check that requires the error to be static. I prefer using the string literal where we are returning the string rather than defining them as consts.